src/node.rs

```diff
@@ -208,17 +250,20 @@ impl Sprawl {
         Ok(&self.forest.nodes[id].layout)
     }
 
+    /// Indicates that the layout of this node and its children must be recomputed
```

Comment:
This sounds to me like it would be something passive and not an action, but I struggle to find a better formulation. Maybe changing it to imperative tense would be better? Or otherwise maybe something like this:

```suggestion
    /// Marks the layout computation of this node and its children as outdated
```